tools/wpt/browser.py

```diff
@@ -935,9 +935,14 @@ def find_binary(self, venv_path=None, channel=None):
                 suffix = " " + channel.capitalize()
             return f"/Applications/Google Chrome{suffix}.app/Contents/MacOS/Google Chrome{suffix}"
         if uname[0] == "Windows":
-            path = os.path.expandvars(r"$SYSTEMDRIVE\Program Files (x86)\Google\Chrome\Application\chrome.exe")
-            if not os.path.exists(path):
-                path = os.path.expandvars(r"$SYSTEMDRIVE\Program Files\Google\Chrome\Application\chrome.exe")
```

Comment:
This was done by #25467 because of migration from `Program Files (x86)` to `Program Files`. `$PROGRAMFILES` should now cover that.